ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/CentOS.yml

```diff
@@ -26,14 +26,14 @@
 - name: Import AZUL public key
   rpm_key:
     state: present
-    key: http://repos.azulsystems.com/RPM-GPG-KEY-azulsystems
+    key: https://repos.azulsystems.com/RPM-GPG-KEY-azulsystems
```

Comment:
Seems to have a certificate that may not work in automation - hostname mismatch

```
repos.azulsystems.com uses an invalid security certificate. The certificate is only valid for the following names: *.cloudfront.net, cloudfront.net 
```

Reply:
Yep my bad, back to http you go, will push in a sec